app/controllers/poms_controller.rb

```diff
@@ -10,8 +10,16 @@ def index
     active_poms, inactive_poms = poms.partition { |pom|
       %w[active unavailable].include? pom.status
     }
-    @active_poms = active_poms.map { |p| PomPresenter.new(p) }
-    @inactive_poms = inactive_poms.map { |p| PomPresenter.new(p) }
+
+    if @prison.womens?
+      # poms are split into groups - 1) probation 2) prison & are sorted by last name
+      @active_probation_poms = active_poms.sort_by(&:last_name).select { |pom| pom.position == 'PO' }.map { |po| PomPresenter.new(po) }
```

Comment:
There are methods called prison_officer? and probation_officer? which already do this - no need to surface position

Reply:
In case this is of any interest, there's a nice example of succinctly splitting Prison Officer / Probation Officer in the allocations controller using the `#partition` method:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/32a6a5ef9aa5ee3ecbfdc867bc0452ed534e4822/app/controllers/allocations_controller.rb#L184-L190

Which could be adapted to something like:

```suggestion
      @active_probation_poms, @active_prison_poms = active_poms.sort_by(&:last_name).map { |p| PomPresenter.new(p) }.partition(&:probation_officer?)
```

One advantage of this approach is performance: you only end up sorting and looping through the array once, rather than twice, to end up with two arrays. In this case I expect the arrays are very small, so the performance gain is negligible, but it might also DRY up the code a little.